pkgs/applications/misc/ArchiSteamFarm/update.sh

```diff
@@ -15,15 +13,14 @@ if [[ "$new_version" == "$old_version" ]]; then
   fi
 fi
 
-asf_path=$(pwd)
+asf_path=$PWD
 cd ../../../..
 
-nixpkgs_path=$(pwd)
 if [[ "${1:-}" != "--deps-only" ]]; then
     update-source-version ArchiSteamFarm "$new_version"
 fi
 
 $(nix-build -A ArchiSteamFarm.fetch-deps --no-out-link) "$deps_file"
 
-cd "$asf_path"
-./web-ui/update.sh
+cd "$asf_path"/web-ui/
+./update.sh
```

Comment:
I think it would look a lot more intuative if we could use `exec "$asf_path"/web-ui/update.sh`, not sure if the script depends on the working directory though.